src/module/actor/actions/single-check.ts

```diff
@@ -137,13 +145,15 @@ class SingleCheckActionVariant extends BaseActionVariant {
 
 class SingleCheckAction extends BaseAction<SingleCheckActionVariantData, SingleCheckActionVariant> {
     readonly difficultyClass?: CheckDC | string;
+    readonly modifiers: RawModifier[];
```

Comment:
Using a `RawModifier` instead of an actual `ModifierPF2e` instance since the localization data is not yet available when the bundled actions are initialized and the ModifierPF2e constructor tries to immediately localise the label/name.